src/realm/object-store/c_api/schema.cpp

```diff
@@ -49,6 +49,12 @@ RLM_API realm_schema_t* realm_get_schema(const realm_t* realm)
     });
 }
 
+RLM_API uint64_t realm_get_schema_version(const realm_t* realm)
+{
+    auto& rlm = *realm;
```

Comment:
I think the code should be wrapped in `wrap_err` as this seems to be the case with (almost) all the other calls.

Reply:
@fealebenpae there's a pointer dereference, so an exception can still happen.

Reply:
`schema_version()` is reading a private field so I guess this one can not fail. But then `realm_get_schema` is wrapping the call indeed. Wonder if both of these can be a direct call? And then mark in the header that these can not fail.

Reply:
@danieltabacaru you're right, but if we fail to dereference a pointer there isn't any kind of usable error we can return to developers. If things ever get so dire that `*realm` throws better let the whole program crash, IMO.

Reply:
@fealebenpae and I have written the comment at the same time. Just in case you are wondering why I am writing the same thing. I guess GH does not auto refresh or something.